client/src/UnconfiguredDashboard.js

```diff
@@ -4,20 +4,21 @@ function UnconfiguredDashboard() {
   return (
     <div>
       <p>
-        Flight Console Service allows you to access an interactive terminal
-        console session running on your cluster from the comfort of your
-        browser.
+        The Flight File Manager Service allows you to manage your files on
```

Comment:
Here and in the blurb, the app is being referred to as File Manager, but elsewhere I've changed the app name to Flight Files